ui/index.d.ts

```diff
@@ -1056,3 +1055,24 @@ export function createAzureSubscriptionClient<T extends IAddUserAgent>(
  * Multiple APIs with different versions can be supplied, but ideally a single backwards-compatible API is all that's necessary.
  */
 export function createApiProvider(azExts: AzureExtensionApi[]): AzureExtensionApiProvider;
+
+// tslint:disable-next-line:max-classes-per-file
+
+/**
+ * Wraps the vscode.OutputChannel to add timestamps to all messages by default.
+ * appendLine accepts optional parameters to override the date and to prepend the message with the resourceName
+ * Use append if you don't want timestamps for that output message
+ */
+export declare class AzureOutputChannel implements OutputChannel {
```

Comment:
Don't you need to use this in the ui package, too? For the resource group step at least? I'm just curious how you plan to make this work with registerUIExtensionVariables